Documentation/k8s-pre-reqs.md

```diff
@@ -66,7 +66,32 @@ Environment="RKT_OPTS=--volume modprobe,kind=host,source=/usr/sbin/modprobe \
 ...
 ```
 
-Note that the kubelet also requires access to the userspace `rbd` tool that is included only in hyperkube images tagged `v1.3.6_coreos.0` or later.
+Note that the kubelet also requires access to the userspace `rbd` tool that is included only in hyperkube images tagged `v1.3.6_coreos.0` or later. See next section on how to deploy `rbd`.
+
+### Ceph and RBD utilities installed on the nodes
+
+The Kubernetes kubelet shells out to system utilities to mount Rook volumes. This means that every Kubernetes host must have these utilities installed. This requirement extends to the control plane, since there may be interactions between kube-controller-manager and the Ceph cluster. Login to each Kubernetes host where Kubelet runs and execute the following:
+
+For Debian-based distros:
+
+```
+apt-get install ceph-fs-common ceph-common
+```
+
+For Redhat-based distros:
+
+```
+yum install ceph
+```
+
+For Other linux distro that don't have an explicit package manager, such as CoreOS, you can use a container with the ceph utilities deployed that you can leverage. To deploy the containers on your hosts, do the following:
```

Comment:
grammer nit: Still need to fix the beginning of this sentence, change it to "For other Linux distros",  i.e., don't capitalize "other", capitalize "Linux" and make "distros" plural.

Also, the end of the sentence is a bit awkward, I'd just remove "deployed that you can leverage" and leave it simply: "... you can use a container with the ceph utilities.  To deploy the containers..."